export.py

```diff
@@ -2061,7 +2062,7 @@ def export_object_attributes(ri, scene, ob, visible_objects):
     obj_groups_str = '*'
     for obj_group in scene.renderman.object_groups:
         if ob.name in obj_group.members.keys():
-            obj_groups_str += ',' + obj_group.name
+            obj_groups_str = obj_group.name
```

Comment:
Really?  You should be able to have a comma separated list of object groups
in rib.  I could possible be wrong about that but I'm 90% sure.


On Mon, May 2, 2016 at 10:33 AM, jdent02 <notifications@github.com> wrote:

> In export.py
> <https://github.com/bsavery/PRMan-for-Blender/pull/251#discussion_r61772008>
> :
>
> > @@ -2061,7 +2062,7 @@ def export_object_attributes(ri, scene, ob, visible_objects):
> >      obj_groups_str = '*'
> >      for obj_group in scene.renderman.object_groups:
> >          if ob.name in obj_group.members.keys():
> > -            obj_groups_str += ',' + obj_group.name
> > +            obj_groups_str = obj_group.name
>
> Gah I didn't even think of that. Problem was when that '*' was in the .rib
> the object group didn't register and I got a black render. I'll play around
> with it a bit more.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/bsavery/PRMan-for-Blender/pull/251/files/59c2821e6ba4bf295f8ed20ca65a1c7e4dc01ac4#r61772008>
>



-- 
brian.savery@gmail.com
508-274-8700